awx_collection/tools/roles/template_galaxy/templates/README.md.j2

```diff
@@ -54,6 +54,7 @@ These can be specified via (from highest to lowest precedence):
  - direct module parameters
  - environment variables (most useful when running against localhost)
  - a config file path specified by the `tower_config_file` parameter
+ - a config file at `./tower_cli.cfg`, i.e. in the current directory
```

Comment:
looking at the code more carefully, it looks like we walk the local directory and check for a tower_cli.cfg file there

for example, if my local directory is /home/sbf/awx, the code will check for
`/home/tower_cli.cfg`
`/home/sbf/tower_cli.cfg`
`/home/sbf/awx/tower_cli.cfg`

with the last one being highest priority

It looks like we never updated these docs to reflect that behavior though

https://github.com/fosterseth/awx/blob/69102cf26565ea6930a28965746831f3cfa08a69/awx_collection/plugins/module_utils/controller_api.py#L170

maybe we can point that out here?